p2p_wallet/UIKit/ViewModels/TransactionsViewModel.swift

```diff
@@ -100,6 +101,7 @@ class TransactionsViewModel: BEListViewModel<SolanaSDK.ParsedTransaction> {
             }
             .do(
                 afterSuccess: { [weak self] transactions in
+                    self?.walletsRepository.reload()
```

Comment:
This is not so nice way to do that, let discuss

Reply:
Can we trigger somehow on pull to refresh event from CollectionView class?